awx/main/utils/licensing.py

```diff
@@ -315,6 +324,7 @@ def is_appropriate_sat_sub(self, sub):
         return True
 
     def is_appropriate_sub(self, sub):
+        return True
```

Comment:
:eyes:  SonarCloud reported that the rest of the method is unreachable. Which... yeah